src/cli/src/cluster/install/k8.rs

```diff
@@ -1,15 +1,20 @@
-use std::process::Command;
 use std::io::Error as IoError;
 use std::io::ErrorKind;
-use super::*;
+use std::process::Command;
+
 use fluvio_cluster::ClusterInstaller;
+use fluvio::config::TlsPolicy;
+use super::*;
 
 pub async fn install_core(opt: InstallCommand) -> Result<(), CliError> {
+    let (client, server): (TlsPolicy, TlsPolicy) = opt.tls.into();
+
     let mut builder = ClusterInstaller::new()
         .with_namespace(opt.k8_config.namespace)
         .with_group_name(opt.k8_config.group_name)
         .with_spu_replicas(opt.spu)
-        .with_save_profile(!opt.skip_profile_creation);
+        .with_save_profile(!opt.skip_profile_creation)
+        .with_tls(client, server);
```

Comment:
need to unit test to ensure InstallCommand is properly translated into ClusterInstall options

Reply:
I'm not sure there's an easy way to do this with the current dependency structure and visibility modifiers. `ClusterInstaller` keeps it's fields private in order to keep implementation freedom for the future, and we don't want to make a two-way dependency by making the Client link to the CLI.